### ISSUE TITLE
Not able to set global errorHandler

### ISSUE THREAD

**Issue by nford88** [nuxt/bridge#685]
Nuxi 3.0.0-rc.12
RootDir: /home/projects/github-rya1ep-up6wk9
Nuxt project info: SSR Nuxt Edge App

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.14.2`
- Nuxt Version:     `2.16.0-27720022.54e852f`
- Nitro Version:    `0.6.1`
- Package Manager:  `npm@7.17.0`
- Builder:          `webpack`
- User Config:      `typescript`, `ssr`, `plugins`, `buildModules`, `modules`, `build`, `serverHandlers`, `devServerHandlers`, `bridge`
- Runtime Modules:  `-`
- Build Modules:    `()`, `@nuxt/bridge@3.0.0-27830071.32c4527`
------------------------------

**Nuxt Bridge and SSR App** 

I can also recreate the issue of @dabrowiecki https://github.com/nuxt/nuxt.js/issues/15078 . I have a feeling it is an issue with Nuxt bridge as you do not get that error in 

Please see my slimdown live recreation  using @danielroe errorHandler plugin suggestion which does not trigger in Vue errorHandler on error.

https://stackblitz.com/edit/github-rya1ep-up6wk9?file=pages/index.vue

Force an internal error with asyncData method in index.vue (uncomment lines 48 and 49 ) check terminal. 

```
[nuxt] [request error] can't access property "dummy", test is null

[nitro] [dev] [unhandledRejection] [asyncData@file:///home/projects/github-rya1ep-up6wk9/.nuxt/dist/server/pages/index.js:46:5
promisify@file:///home/projects/github-rya1ep-up6wk9/.nuxt/dist/server/server.js:1987:15
./.nuxt/server.js/__webpack_exports__.default/asyncDatas<@file:///home/projects/github-rya1ep-up6wk9/.nuxt/dist/server/server.js:1644:82
./.nuxt/server.js/__webpack_exports__.default@file:///home/projects/github-rya1ep-up6wk9/.nuxt/dist/server/server.js:1639:51
]
```
This should return the text we stipulate in the errorHandler but it does not.

On a a side note to this issue, I do not think consola plays nicely with Nuxt/Nuxt Bridge. 

I am using consola for global logging attaching to the stdout/stderr process. It only seems to work when directly invoked consola in my examples within asyncData rather than catching all console messages. 

Also when I build and serve the SSR application, the Nitro logging defaults to verbose instead of  attaching consola. When using dev server all Nitro logging is done via consola. Strange

I can post the specific logging issue in another topic, for now I would like to try and get the Vue Error Handler resolved first.

Many thanks!

_Originally posted by @nford88 in https://github.com/nuxt/nuxt.js/issues/15078

**Comment by nford88** [comment]
Is there any update on this?

**Comment by danielroe** [comment]
resolved in #783
